pkg/template/registry/templateinstance/strategy.go

```diff
@@ -55,9 +55,19 @@ func (templateInstanceStrategy) PrepareForCreate(ctx apirequest.Context, obj run
 	templateInstance := obj.(*templateapi.TemplateInstance)
 
 	if templateInstance.Spec.Requester == nil {
-		user, _ := apirequest.UserFrom(ctx)
-		templateInstance.Spec.Requester = &templateapi.TemplateInstanceRequester{
-			Username: user.GetName(),
+		user, ok := apirequest.UserFrom(ctx)
+		if ok {
```

Comment:
that occurred to me too, but how to do it was not obvious ... `PrepareForCreate` does not return err or and ErrorList.

I do see that the Context interface has an Err() method and my preliminary searches are finding hits around canceling the context.  Perhaps that is what should be done, but I'm still looking for a precise example for guidance.  Of course any pointers to save me search time are appreciated.